app/views/tags/_tag.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="tag-box">
+  <a href="/tags/<%= tag[:id] %>" class="discourse-tag simple"><%= tag[:text] %></a>
```

Comment:
That link won't work when using subfolder instance.

Reply:
I have tried to fix it, let me know if it works :slightly_smiling_face:  @ZogStriP